src/Processors/Transforms/Streaming/TumbleHelper.cpp

```diff
@@ -0,0 +1,111 @@
+#include <Processors/Transforms/Streaming/TumbleHelper.h>
```

Comment:
Let's rename all Helper in the namespace to `Window` like `TumbleWindow`, `HopWindow` and `SessionWindow`